### DISCUSSION TITLE
macOS AE版本 24.6.0 （Build 69）插件会大概率报错

### DISCUSSION THREAD

**Discussion by wyfang** [Tencent/libpag#2482] [General] [answered]
报错后发现只要将'/Library/Application Support/Adobe/Common/Plug-ins/7.0/MediaCore'里的PAGExporter删除就正常。

**Comment by jinwuwu001** [comment, marked as answer]
该问题已与用户沟通，用户怀疑是AE的问题，因此不需要再跟进处理，现关闭该问题
